src/views/VisitsView.vue

```diff
@@ -10,9 +10,23 @@
       @toggled-search-panel="searchPanelIsCollapsed = !searchPanelIsCollapsed"
     />
     <div :class="['search-content-wrapper']">
+      <b-alert
+        v-model="showInfo"
+        variant="info"
+        dismissible
+        @dismissed="infoDismissed"
+      >
+        {{ infoMessage }}
+      </b-alert>
+
       <div class="search-results">
         <div class="results-summary">
-          <h1>Visits</h1>
+          <div style="display:flex;">
+            <h1 style="flex-grow: 100;">Visits</h1>
+            <div style="align-self: flex-end;">
+              <b-button variant="link" @click="showInfo = true">Help</b-button>
```

Comment:
What about using a Font Awesome question mark icon? There's already two variants being imported in fontAwesomeIcons.js so it should be easy to try out.